<launch>
      <arg name="enable_laser" default="true"/>

      <!--roslaunch husky_gazebo husky_playpen.launch-->
      <include file="$(find husky_gazebo)/launch/husky_playpen.launch"/>
      <param name="robot_description" command="$(find xacro)/xacro '$(find husky_description)/urdf/husky.urdf.xacro' laser_enabled:=$(arg enable_laser)"/>

      <!--Parameters for goal and maximum velocity-->
      <param name="/rob_midterm_dsmb/destination_x" type="double" value="3.0" />
      <param name="/rob_midterm_dsmb/destination_y" type="double" value="1.5" />
      <param name="/rob_midterm_dsmb/velocity_max" type="double" value="1" />

      <!--Start nodes-->
      <node name="avoidance_husky" pkg="rob_midterm_dsmb" type="avoidance_husky.py"/>
      <node name="merge_scans" pkg="rob_midterm_dsmb" type="merge_scans.py"/>
      <node name="nav_husky" pkg="rob_midterm_dsmb" type="nav_husky.py"/>
</launch>